src/commands/cmd_json.cc

```diff
@@ -52,7 +52,37 @@ class CommandJsonGet : public Commander {
   }
 };
 
+class CommandJsonArrAppend : public Commander {
+ public:
+  Status Execute(Server *svr, Connection *conn, std::string *output) override {
+    redis::Json json(svr->storage, conn->GetNamespace());
+
+    std::vector<uint64_t> result_count;
+
+    auto s = json.ArrAppend(args_[1], args_[2], {args_.begin() + 3, args_.end()}, result_count);
+    if (!s.ok()) return {Status::RedisExecErr, s.ToString()};
+
+    if(result_count.empty()) {
+      result_count.emplace_back(0);
+    }
+
+    std::vector<std::string> result_values;
+    result_values.reserve(result_count.size());
+    for(auto c : result_count) {
```

Comment:
```suggestion
    for(uint64_t c : result_count) {
```

Since `uint64_t` is not a too complex type, maybe we'd better using literal rather than auto